index.html

```diff
@@ -6,11 +6,81 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
+    <link rel="stylesheet" href="./style.css">
     <!-- Add a link to your css file here -->
 </head>
+<div class="grid-container">
+    <header class="header">
+        <div class="company-logo">
+            <img class="company-logo-img" src="./images/cupcake-icon.png" alt="company logo">
+        </div>
+        <div class="hamburger-motto">
+            <div class="hamburger">
+                <span class="bar"></span>
```

Comment:
there are lots of free icons you can get online for commonly used icons such as a hamburger, e.g. [Font Awesome](https://fontawesome.com/v5.15/icons?d=gallery&p=2)